File: app/src/main/java/com/marcoscg/materialtoastsample/MainActivity.java
package com.marcoscg.materialtoastsample;

import android.graphics.Color;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import com.marcoscg.materialtoast.MaterialToast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void toastDef(View v) {
        Toast.makeText(this, "Hello, I'm a default toast!", Toast.LENGTH_SHORT).show();
    }

    public void toastMat(View v) {
        MaterialToast.makeText(this, "Hello, I'm a material toast!", R.mipmap.ic_launcher, Toast.LENGTH_SHORT).show();
    }

    public void toastMatCol(View v) {
//        MaterialToast.makeText(this, "Hello, I'm a material toast!",
//                R.mipmap.ic_launcher, Toast.LENGTH_SHORT).setBackgroundColor(Color.RED).show();

        new MaterialToast(this)
                .setMessage("Hello, I'm a material toast!")
                .setIcon(R.mipmap.ic_launcher)
                .setDuration(Toast.LENGTH_SHORT)
                .setBackgroundColor(Color.RED)
                .show();
    }
}

File: app/src/main/java/com/marcoscg/materialtoastsample/MainActivity.kt
package com.marcoscg.materialtoastsample

import android.graphics.Color
import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.marcoscg.materialtoast.MaterialToast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun toastDef(v: View?) {
        Toast.makeText(this, "Hello, I'm a default toast!", Toast.LENGTH_SHORT).show()
    }

    fun toastMat(v: View?) {
        MaterialToast.makeText(this, "Hello, I'm a material toast!", R.mipmap.ic_launcher, Toast.LENGTH_SHORT).show()
    }

    fun toastMatCol(v: View?) {
//        MaterialToast.makeText(this, "Hello, I'm a material toast!",
//                R.mipmap.ic_launcher, Toast.LENGTH_SHORT).setBackgroundColor(Color.RED).show();
        MaterialToast(this)
                .setMessage("Hello, I'm a material toast!")
                .setIcon(R.mipmap.ic_launcher)
                .setDuration(Toast.LENGTH_SHORT)
                .setBackgroundColor(Color.RED)
                .show(v)
    }
}